guides/relay/connections.md

```diff
@@ -87,9 +87,9 @@ end
 Now, you can use `PostConnectionWithTotalCountType` to define a connection with the "totalCount" field:
 
 ```ruby
-AuthorType = GraphQL::ObjectType.define do
+class AuthorType < GraphQL::Schema::Object
   # Use the custom connection type:
-  connection :posts, PostConnectionWithTotalCountType
+  field :posts, PostConnectionWithTotalCountType, null: false
```

Comment:
When the type name does not end with `Connection` then you need to pass `connection: true` option.

Reply:
`PostConnectionWithTotalCountType` is being created as a connection using the `defined_connection` method. As these types will be postfixed with `Connection, I'm not sure if you need this connection argument as the lib can detect that based on that connection return type.

Reply:
Connection is just a type with connection specific fields. You need to configure the field so the field resolver could be wrapped with connection resolver. 
And currently it is done by connection option or by the object type name. 

That's what current documentation says (http://graphql-ruby.org/schema/class_based_api.html#connection-fields--types):
>>>
There is no connection(...) method. Instead, connection fields are inferred from the type name.
If the type name ends in Connection, the field is treated as a connection field.

Reply:
Oh, thanks for the correction !